docs/stdlib/cfg.rst

```diff
@@ -0,0 +1,148 @@
+.. _ref_std_cfg:
+
+======
+Config
+======
+
+The ``cfg`` module contains a set of types and scalars used for configuring
+EdgeDB.
+
+
+.. list-table::
+  :class: funcoptable
+
+  * - **Type**
+    - **Description**
+  * - :eql:type:`cfg::AbstractConfig`
```

Comment:
Querying `AbstractConfig` currently raises an ISE (which is a bug, but still). I recommend just documenting `cfg::Config` which acts like a view -- it reflects whatever setting values are set in your instance/db/session maintaining the correct resolution order.